clang/tools/checked-c-convert/IterativeItypeHelper.h

```diff
@@ -0,0 +1,39 @@
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
```

Comment:
Please use the new LLVM license header for all source files. You can find it here: https://llvm.org/foundation/relicensing.

Reply:
Done. Resolved by https://github.com/microsoft/checkedc-clang/pull/826/commits/3efa40851f41eaec46c38ed056f0b82ee8355c48